poc/idpf.sage

```diff
@@ -117,6 +119,31 @@ def test_idpf(Idpf, alpha, level, prefixes):
                 alpha, prefix, level, got, want))
 
 
+def gen_test_vec(Idpf, alpha, test_vec_instance):
+    beta_inner = [[Idpf.FieldInner(1)] * Idpf.VALUE_LEN] * (Idpf.BITS-1)
+    beta_leaf = [Idpf.FieldLeaf(1)] * Idpf.VALUE_LEN
+    (public_share, keys) = Idpf.gen(alpha, beta_inner, beta_leaf)
+
+    printable_beta_inner = [
+        [ elem.as_unsigned() for elem in value ] for value in beta_inner
+    ]
+    printable_beta_leaf = [ elem.as_unsigned() for elem in beta_leaf ]
```

Comment:
These should be stringified to avoid JSON library compatibility issues

Reply:
Done.